batch/indexer_Position_Coupon.py

```diff
@@ -162,8 +156,13 @@ def __get_contract_list(self, db_session: Session):
                     contract_name="IbetCoupon",
                     address=listed_token.token_address
                 )
-                self.token_list.append(token_contract)
-                self.token_address_list.append(token_contract.address)
+                synced_block_number = self.__get_idx_position_block_number(
+                    db_session=db_session,
+                    token_address=listed_token.token_address
+                )
+                block_from = synced_block_number + 1
+                token = self.TargetToken(token_contract, block_from)
+                self.token_list.append(token)
```

Comment:
Load latest synced block number of each token. 
And store the number as instance value.